README.md

```diff
@@ -1,63 +1,79 @@
-function, however, developers will be able to determine the MeshID of
-their superpeer(s) and use those to let the Mesh determine the best way
-to reach the superpeer(s).
-
-Don't forget when you clone the repo to update your credentials from 
-[https://developer.rightmesh.io] in the [build.gradle](build.gradle) 
-file.
-
-# Setting up
-If you want to run the parity client directly, run the `configure.sh`
-script. This will install parity from .deb (along with all dependencies)
-and start it up.
+# RightMesh Superpeer
 
-Otherwise you can uncomment the last few lines in the 
-build.gradle script to run parity in vagrant. If you wish to run it
-in vagrant, you'll need to install vagrant and virtualbox. This will
-install the parity wallet within the vm so you don't accidentally
-clobber your local wallet.
+This minimalist RightMesh "Superpeer" implementation provides two functionalities: enabling the forwarding of data packets from one geographically separate mesh to another, and facilitating transactions involving RightMesh Tokens. The second functionality includes recording transactions to the Ethereum blockchain.
+
+## Running the Superpeer
+
+Superpeer is a Gradle application - it can be built with `gradlew build`, and a binary can be generated with `gradlew installDist`.
+
+If run without arguments Superpeer polls STDIN for input - typing `exit` will shut down RightMesh and stop the application. Unless you are developing/debugging, you will likely want to run Superpeer with the `-h | --headless` flag, which doesn't poll for input and responds to SIGINT signals (e.g. can be killed cleanly with `Ctrl+C` or task managers).
+
+## Connecting to your Superpeer
+
+By default RightMesh devices with internet connections will connect to a Superpeer operated by RightMesh at `research.rightmesh.io`. In the future the goal is to have a network of Superpeers, with Superpeers implemented and operated by both RightMesh and community members.
+
+If you would like to have devices use your Superpeer instance for testing, you can hard-code its address into a RightMesh application to make it use your instance instead of the production network. You can do this by passing the address as a parameter to a special MeshManager constructor that is available in the developer version of the library. Note that this will never be an option in production - all RightMesh applications will use the same network, so that the owner of the device can specify their preferences for where/when/how to connect to Superpeers.
 
-Todo: detect in gradle or use a set variable to specify whether we are
-running on a local machine or AWS instance. Also add an option to set
-a remote parity machine (if the superpeer wants to separate funciton).
+If you would like to send data to a Java application that makes use of RightMesh in a production application, you will need to find out its MeshID and send data to it through the RightMesh library.
```

Comment:
Not sure why this needs to be distinguished from an other RightMesh node - a java node and and android behave the same way in this respect.

Reply:
This was a holdover from Superpeer doubling as say servers, which might become a thing in the future with App Superpeers.

So it's here to explain "if you want a node to act like a centralized 'server' in a traditional sense and have it run on a server, use RightMesh not a Superpeer."

Reply:
Like that a MeshID is the way to talk to it, not it's IP address.